gulpfile.js

```diff
@@ -421,3 +423,10 @@ gulp.task('imagemin', () => {
 });
 
 gulp.task('minify', ['jsmin', 'svgmin', 'imagemin']);
+
+gulp.task('precompile-templates', () =>
```

Comment:
This had some issues on occasion – I had to restart my server quite often while working.

Reply:
Hm, what sort of errors?

Reply:
Only time I've been able to reproduce errors is when I make multiple quick changes in a row, before compilation has time to complete.